big_tests/tests/sm_SUITE.erl

```diff
@@ -124,6 +127,9 @@ parallel_unacknowledged_message_hook_cases() ->
      unacknowledged_message_hook_resume,
      unacknowledged_message_hook_filter].
 
+ping_timeout_cases() ->
```

Comment:
do we really need a group for just one testcase?

Reply:
I believed that introducing a separate group would be an optimal solution, given that the stanzas that are sent by `mod_ping` tend to create issues in other tests